el-gr/json-gr.html.markdown

```diff
@@ -0,0 +1,53 @@
+---
+language: json
+filename: json-gr.html.markdown
+contributors:
+  - ["Athanasios Emmanouilidis", "https://github.com/athanasiosem"]
+---
```

Comment:
Please change this to 

```
language: json
filename: json-gr.html.markdown
contributors:
  - ["Anna Harren", "https://github.com/iirelu"]
  - ["Marco Scannadinari", "https://github.com/marcoms"]
  - ["himanshu", "https://github.com/himanshu81494"]
  - ["Michael Neth", "https://github.com/infernocloud"]
  - ["Athanasios Emmanouilidis", "https://github.com/athanasiosem"]
translators:
  - ["Athanasios Emmanouilidis", "https://github.com/athanasiosem"]
lang: el-gr
```

Reply:
The original contributors list needs to be present and your name would appear in both it and the and translators list. Also the `lang` line needs to be present for the site builder to correctly identify this doc as a translation of an existing one and not a standalone doc.